learn/user-guide/publishing-modules-to-ballerina-central.md

```diff
@@ -19,7 +19,7 @@ redirect_from:
 
 ## The CLI Command
 
-Pushing a package uploads it to [Ballerina Central](https://central.ballerina.io/).
+Pushing a package uploads it to [Ballerina Central](https://swanlake.central.ballerina.io/).
```

Comment:
https://swanlake.central.ballerina.io/ is a temporary URL we will be migrating https://central.ballerina.io/ with those change so lets keep the URL as it is